content/docs/1.4.0/advanced-resources/rwx-workloads.md

```diff
@@ -1,60 +1,69 @@
 ---
-title: Support for ReadWriteMany (RWX) workloads (Experimental Feature)
+title: ReadWriteMany (RWX) Volume
 weight: 4
 ---
 
-Longhorn natively supports RWX workloads, by exposing a regular Longhorn volume via a NFSv4 server (share-manager).
+Longhorn supports ReadWriteMany (RWX) volumes by exposing regular Longhorn volumes via NFSv4 servers that reside in share-manager pods.
 
-The following diagram shows how the RWX support works:
 
-{{< figure src="/img/diagrams/rwx/rwx-native-architecture.png" >}}
+# Introduction
 
-For each actively in use RWX volume Longhorn will create a `share-manager-<volume-name>` Pod in the `longhorn-system` namespace.
+For each actively in use RWX volume Longhorn will create a `share-manager-<volume-name>` Pod in the `longhorn-system` namespace. This Pod is responsible for exporting a Longhorn volume via a NFSv4 server that is running inside the Pod. There is also a service created for each RWX volume, and that is used as an endpoint for the actual NFSv4 client connection.
 
-This Pod is responsible for exporting a Longhorn volume via a NFSv4 server that is running inside the Pod.
-
-There is also a service created for each RWX volume, and that is used as an endpoint for the actual NFSv4 client connection.
+{{< figure src="/img/diagrams/rwx/rwx-arch.png" >}}
 
 # Requirements
 
-To be able to use RWX volumes, each client node needs to have a NFSv4 client installed.
+It is necessary to meet the following requirements in order to use RWX volumes.
 
-For Ubuntu you can install a NFSv4 client via:
+1. Each NFS client node needs to have a NFSv4 client installed.
+     - For Debian based distros you can install a NFSv4 client via:
```

Comment:
Can add a SUSE/openSUSE example. Also, there is a help manifest users can leverage.

Reply:
Updated. Thank you.